src/iotjs_binding.h

```diff
@@ -108,6 +108,11 @@ jerry_value_t iotjs_jhelper_eval(const char* name, size_t name_len,
                                  const uint8_t* data, size_t size,
                                  bool strict_mode);
 
+/* Note:
+ *      Defines started with underscores should not be used
+ *      outside of this header.
```

Comment:
How about using `#undef`?

Reply:
Good idea. I'd look better with `#undef`. I'll update the patch.

Reply:
Sorry. This case cannot do because it is a recursive definition.